dist/profile/files/bind/jenkins.io.zone

```diff
@@ -46,14 +46,13 @@ usage       3600    IN    A    52.204.62.78   ; jenkins-usage
 nginx.azure   3600  IN    A       13.68.19.38 ; kubernetes service: namespace:nginx-ingress name:nginx
 plugins.azure 3600  IN    CNAME   nginx.azure    ; used to test plugins.jenkins.io migration
 repo.azure    3600  IN    CNAME   nginx.azure    ; Proxy cache in front of repo.jenkins-ci.org
-beta.accounts 3600  IN    CNAME   nginx.azure    ; Must be deleted when accounts is a CNAME for nginx.azure
+accounts      3600  IN    CNAME   nginx.azure    ; Must be deleted when accounts is a CNAME for nginx.azure
```

Comment:
Looks like the comment needs to be removed.

Reply:
Nice catch

Reply:
#842 remove this comment